src/lib/protocols/proof/messages/messages.ts

```diff
@@ -0,0 +1,6 @@
+export enum MessageType {
+  PresentProof = ' "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/present-proof/1.0/propose-presentation"',
+  ProposePresentation = 'did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/present-proof/1.0/request-presentation',
```

Comment:
Same here `RequestPresentation` instead of `ProposePresentation`

Reply:
Changed